Makefile

```diff
@@ -71,7 +71,7 @@ libp2p_helper:
 
 genesis_ledger: ocaml_checks
 	$(info Building runtime_genesis_ledger)
-	ulimit -s 65532 && (ulimit -n 10240 || true) && env MINA_COMMIT_SHA1=$(GITLONGHASH) dune exec --profile=$(DUNE_PROFILE) src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe -- --genesis-dir $(GENESIS_DIR)
+	((ulimit -s 65532 || true) || true) && (ulimit -n 10240 || true) && env MINA_COMMIT_SHA1=$(GITLONGHASH) dune exec --profile=$(DUNE_PROFILE) src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe -- --genesis-dir $(GENESIS_DIR)
```

Comment:
it'd be good to have someone from infra look at this change :D I admit I don't really know the consequences of these ulimit lines or even why they're here in the first place (seems odd to me)

Reply:
it's a relic of a very old time